PreMailer.Net/PreMailer.Net/PreMailer.cs

```diff
@@ -245,9 +245,14 @@ private bool DomainMatch(string domain, string url)
 		/// Returns the blocks of CSS within the documents supported CSS sources.<para/>
 		/// Blocks are returned in the order they are declared within the document.
 		/// </summary>
-		private IEnumerable<string> GetCssBlocks(IEnumerable<ICssSource> cssSources)
+		private List<string> GetCssBlocks(IEnumerable<ICssSource> cssSources)
```

Comment:
```suggestion
		private IEnumerable<string> GetCssBlocks(IEnumerable<ICssSource> cssSources)
```

It should be possible to keep this.